simplq/src/store/asyncActions/deleteToken.js

```diff
@@ -21,8 +21,9 @@ const useDeleteToken = () => {
     const { tokenId } = arg;
     const authedRequest = makeAuthedRequest(RequestFactory.deleteToken(tokenId));
     const response = await authedRequest;
-    dispatch(setInfoPopupMessage('Successfully left queue'));
+    // TODO: remove token from redux store on success
     if (arg.goHome) {
```

Comment:
After adding logic for this and similar logic for adding members to modify the token slice, the page should be much more responsive. Let's do that in a subsequent PR.